testsuite/smoke-test/src/aptos_cli/validator.rs

```diff
@@ -340,6 +352,179 @@ async fn test_onchain_config_change() {
+        transaction_shuffler_type: TransactionShufflerType::SenderAwareV1(32),
+    });
+
+    let update_execution_config_script = format!(
+        r#"
+    script {{
+        use aptos_framework::aptos_governance;
+        use aptos_framework::execution_config;
+        fun main(core_resources: &signer) {{
+            let framework_signer = aptos_governance::get_signer_testnet_only(core_resources, @0000000000000000000000000000000000000000000000000000000000000001);
+            let config_bytes = {};
+            execution_config::set(&framework_signer, config_bytes);
+        }}
+    }}
+    "#,
+        generate_blob(&bcs::to_bytes(&execution_config_with_shuffling).unwrap())
+    );
+
+    cli.run_script(root_cli_index, &update_execution_config_script)
+        .await
+        .unwrap();
+
+    let updated_execution_config: OnChainExecutionConfig = bcs::from_bytes(
+        &rest_client
+            .get_account_resource_bcs::<Vec<u8>>(
+                CORE_CODE_ADDRESS,
+                "0x1::execution_config::ExecutionConfig",
+            )
+            .await
+            .unwrap()
+            .into_inner(),
+    )
+    .unwrap();
+
+    assert_eq!(
+        updated_execution_config.transaction_shuffler_type(),
+        TransactionShufflerType::SenderAwareV1(32)
+    );
+
+    assert_reordering(&mut swarm, true).await;
+}
+
+
+#[tokio::test]
+async fn test_reordering() {
+    let mut swarm = SwarmBuilder::new_local(1)
+        .with_init_config(Arc::new(move |_, config, _| {
+            config.consensus.wait_for_full_blocks_above_pending_blocks = 1;
+            config.consensus.wait_for_full_blocks_above_recent_fill_threshold = 0.1;
+        }))
+        .build()
+        .await;
+
+    assert_reordering(&mut swarm, true).await;
+}
+
+async fn assert_reordering(swarm: &mut dyn Swarm, expected_reordering: bool) {
+    swarm
+        .aptos_public_info()
+        .sync_root_account_sequence_number()
+        .await;
+    let transaction_factory = swarm.aptos_public_info().transaction_factory();
+
+    let clients = swarm.get_all_nodes_clients_with_names();
+
+    let dst = create_and_fund_account(swarm, 10000000000).await;
+
+    let mut accounts = vec![];
+    let mut txns = vec![];
+    for _ in 0..2 {
+        let mut account = create_and_fund_account(swarm, 10000000000).await;
+
+        for _ in 0..5 {
+            let txn = account.sign_with_transaction_builder(
+                transaction_factory.payload(aptos_stdlib::aptos_coin_transfer(dst.address(), 10)),
+            );
+            txns.push(txn);
+        }
+        accounts.push(account);
+    }
+
+    let result = clients[0]
+        .1
+        .submit_batch_bcs(&txns)
+        .await
+        .unwrap()
+        .into_inner();
+    println!("result: {:?}", result);
+    tokio::time::sleep(Duration::from_secs(3)).await;
+
+    wait_for_all_nodes_to_catchup(&clients, Duration::from_secs(30))
+        .await
+        .unwrap();
```

Comment:
can you replace these two with:

for txn in &txns {
    clients[0].1.wait_for_signed_transaction(txn).await.unwrap();
}